src/System.Text.Json/pkg/Microsoft.Bcl.Json.Sources.pkgproj

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <UsingTask TaskName="GenerateSourceWithPublicTypesAsInternal" AssemblyFile="$(PackagingTaskDir)Microsoft.DotNet.Build.Tasks.Packaging.dll" />
+
+  <PropertyGroup>
+    <!-- We need to make sure that all of the surface area on the sources gets rewritten as internal. -->
+    <GenerateInternalTypesSource>true</GenerateInternalTypesSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_AllJsonSources Include="$(MSBuildThisFileDirectory)../src/**/*.cs" />
+    <SourcePackageFiles Include="@(_AllJsonSources)">
+      <PackagePath>contentFiles/cs/netstandard2.0/%(RecursiveDir)%(FileName)%(Extension)</PackagePath>
+    </SourcePackageFiles>
+    <File Include="$(PlaceHolderFile)">
```

Comment:
I'm not sure of the value of this.  Do you really want this package to be referenced on netcoreapp3.0 and do nothing 😕

Reply:
the main idea here is that we don't want somebody that used to depend on this package and upgrades the framework they are targeting to be in a broken state (given that there would be two places where the types would be defined) That is why we wanted to add the placeholder so that if somebody targeting netcoreapp3.0+ would just get a no-op output from this package.